src/docs/perf/rendering/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Improving rendering performance
+description: How to measure and evaluate your app's rendering performance.
+---
+
+Rendering animations in your app is one of the most cited
+topics of interest when it comes to measuring performance. 
+Thanks, in part, to Flutter's Skia engine and its ability
+to quickly create and dispose of widgets, 
+Flutter applications are performant by default,
+so you only need to avoid common pitfalls to achieve
+excellent performance instead of needing to micro-optimize
+with complicated profiling tools.
```

Comment:
I would strike the part about complicated profiling tools, and instead focus on Flutter being fast by default.